### ISSUE TITLE
Consider calculating market cap using coingecko "circulating supply" and not CELO "total supply"

### ISSUE THREAD

**Issue by 0xarthurxyz** [celo-org/blockscout#964]
**TLDR:**

- Our market cap on the [explorer home page](https://explorer.celo.org/mainnet/) uses the "total supply" and not "circulating supply" like Coinmarketcap does
- Defining "circulating supply" is relatively arbitrary, so I presume we opted to keep it simple (more details below)

**More details**:

- Our market cap calculation: $302,793,843 = 679,828,247.85 ([total supply](https://explorer.celo.org/mainnet/token/0x471EcE3750Da237f93B8E339c536989b8978a438/read-proxy)) x $0.445 ([token price](https://www.coingecko.com/en/coins/celo))
- Coinmarketcap calculation: $227,695,737 = 511,790,944 ([circulating supply](https://coinmarketcap.com/currencies/celo/)) x 0.445 ([token price](https://coinmarketcap.com/currencies/celo/))
- I'm not sure whether this calculation is intentional or by mistake on our side, because the GoldToken ERC-20 contract returns the same number `679,828,247.85` for both the `totalSupply()` and `circulatingSupply()` [methods](https://explorer.celo.org/mainnet/address/0xcB8710e072aC4700eE7eD0C63B2f2102366a7a39/contracts#address-tabs).
  <img src="https://github.com/celo-org/blockscout/assets/46296830/74d9738c-f942-4504-9af1-a3dc4ac7a280" width="450">
  <img src="https://github.com/celo-org/blockscout/assets/46296830/de537df0-fe8a-469d-8512-4b7a678c1a3d" width="450">

**Todo**:
- [ ] Discuss whether current market cap calculation is appropriate
- [ ] If necessary, fix calculation to use "circulating supply"

**Comment by Dee297** [comment]
I am trying to figure out what's the current circulating supply. On the stats page, it's possible to know the [total locked Celo](https://rc1-blockscout.celo-testnet.org/mainnet/stats). Is there a way to know what's the current total unlocked Celo in circulation?

**Comment by 0xarthurxyz** [comment]
Good question, I assume it's possible, but I don't have a specific query on hand for that. 

To get the total CELO that is locked you can call the `getTotalLockedGold` function in the [`lockedGold.sol` smart contract](https://github.com/celo-org/celo-monorepo/blob/1c9c844cf57967aa13cb05ccdccd86cb6dcff92f/packages/protocol/contracts/governance/LockedGold.sol#L314).

Alternatively, if you need a time-series, you can use [these two events](https://github.com/celo-org/celo-monorepo/blob/1c9c844cf57967aa13cb05ccdccd86cb6dcff92f/packages/protocol/contracts/governance/LockedGold.sol#L92) to compute the amount of locked gold over time using dune.com.

```solidity
event GoldLocked(address indexed account, uint256 value);
event GoldUnlocked(address indexed account, uint256 value, uint256 available);
```
